optional_components/speech_api/Makefile

```diff
@@ -1,19 +1,28 @@
+SHELL := /bin/bash
+
 .PHONY: tests
 
-# Main test target
 tests: setup-tests run-tests clean-tests
 
-# Test runner
 run-tests:
 	@set -a && source ./tests/test.env && set +a && \
 	python -m pytest -rPQ tests
 
-# Target to create the directory with the correct permissions
 setup-tests:
 	set -a && source ./tests/test.env && set +a && \
 	mkdir -p $$WHISPER_MODEL_DIR && \
-	chmod -R 755 $$WHISPER_MODEL_DIR
+	chmod -R 755 $$WHISPER_MODEL_DIR && \
+	mkdir -p $$PIPER_MODELS_DIR && \
```

Comment:
So we need to download all models just to test? Can't we just mock it ?

Reply:
But if mock it all here then we wouldn't be testing anything as such